runtime/compiler/control/JitDump.cpp

```diff
@@ -168,31 +152,10 @@ static UDATA logStackIterator(J9VMThread *currentThread, J9StackWalkState *walkS
    {
    Trc_JIT_DumpWalkingFrame(currentThread);
 
-   // stop iterating if the walk state is null
-   if (!walkState)
-      {
-      stackWalkEndingBecause("got a null walkState");
-      return J9_STACKWALK_STOP_ITERATING;
-      }
```

Comment:
No, the `walkState` is passed in by the caller `runJitdump` and the same pointer is used throughout the stack walk. This object is created on the stack in `runJitdump`. There is absolutely no need to to null check it. It will never be null.